unit_tests/test_dbprocessing.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
This strikes me as a real source of misery. Is there a particular reason for it?

Reply:
Yep,  the "shasum" return a binary Python 3 format `b'abcd'`.   I think this was why, I just want a string on both Python 2 and Python 3

Reply:
I don't think unicode_literals is going to change the return value from subprocess; I'd expect that to always be a byte object. So maybe we drop this based on the outcome of the hash function calculation?